### PR TITLE
[BEAM-3900] Allow for Beam windowing to be set through Euphoria API.

### PR DESCRIPTION
**Pull request by VaclavPlajt**
`cz.seznam.euphoria.beam.window.BeamWindowing` Contains all components needed to define Beam's windowing. It also implements Euphoria windowing so it can be used in Eupohora API without changes.

Open points:
* Do we still want to supports Euphria's windowing? Removing it might further clean API.

### CODE REVIEW

File: sdks/java/extensions/euphoria/euphoria-beam/src/main/java/cz/seznam/euphoria/beam/window/UnsupportedWindow.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package cz.seznam.euphoria.beam.window;
```
Comment (dmvk) [inline sdks/java/extensions/euphoria/euphoria-beam/src/main/java/cz/seznam/euphoria/beam/window/UnsupportedWindow.java:18]: missing license header

**Review by dmvk — APPROVED** [review]
LGTM 👍 I reckon we should completely remove Euphoria windowing in near future. I have just one minor comment regarding missing license, but that should be fixed in "build integration ticket".

File: sdks/java/extensions/euphoria/euphoria-beam/src/main/java/cz/seznam/euphoria/beam/window/UnsupportedWindow.java (hunk 1 above)
Comment (VaclavPlajt) [reply sdks/java/extensions/euphoria/euphoria-beam/src/main/java/cz/seznam/euphoria/beam/window/UnsupportedWindow.java:18]: fixed

**Review by mareksimunek — APPROVED** [review]
good job

**Pull request merged**